src/Chapter1.hs

```diff
@@ -568,7 +573,9 @@ True
 >>> isVowel 'x'
 False
 -}
-isVowel c = error "isVowel: not implemented!"
+
+isVowel :: Char -> Bool
+isVowel c = c `elem` ['a', 'e', 'i', 'o','u']
```

Comment:
Absolutely correct! 👍🏼 

One note in here, that sometimes, `elem` could be slower than the explicit pattern matching. I remember there were some benchmarks on one particular case, that showed how moving to pattern matching on each case separately drastically decrease time 🐎

Reply:
```
isVowelElem :: Char -> Bool
isVowelElem c = c `elem` ['a', 'e', 'i', 'o', 'u']

isVowelCase :: Char -> Bool
isVowelCase c =
  case c of
    'a' -> True
    'e' -> True
    'i' -> True
    'o' -> True
    'u' -> True
    _ -> False

benchIsVowel :: (Char -> Bool) -> IO Integer
benchIsVowel f = do
  start <- getCPUTime
  let r = map f $ take 100000 $ cycle ['a' .. 'z']
  end <- r `deepseq` getCPUTime
  return (end - start)
```
```
*Chapter1> a <- (`div` 100) . sum <$> mapM (const $ benchIsVowel isVowelElem ) [0..99]
*Chapter1> b <- (`div` 100) . sum <$> mapM (const $ benchIsVowel isVowelCase ) [0..99]
*Chapter1> a
25075167500
*Chapter1> b
15248074970
*Chapter1> fromIntegral b / fromIntegral a
0.6080946406439758
```

With this toy benchmark it seems, the variant using pattern match takes only 60% of the time of the `elem` variant

I applied the suggestion in https://github.com/ccntrq/learn4haskell/pull/1/commits/ce891f464047919f418210656b1d7cf115d48060